OfficeUIFabric/src/main/java/com/microsoft/officeuifabric/persona/AvatarView.kt

```diff
@@ -45,18 +47,28 @@ open class AvatarView : AppCompatImageView {
         if (avatarImageResourceId > 0 && resources.getResourceTypeName(avatarImageResourceId) == "drawable")
             avatarImageDrawable = styledAttrs.getDrawable(R.styleable.AvatarView_avatarImageDrawable)
 
+        val avatarBackgroundColorResourceId = styledAttrs.getResourceId(R.styleable.AvatarView_backgroundColor, 0)
+        if (avatarBackgroundColorResourceId > 0 && resources.getResourceTypeName(avatarBackgroundColorResourceId) == "color")
+            backgroundColor = ContextCompat.getColor(context, avatarBackgroundColorResourceId)
+
         styledAttrs.recycle()
     }
 
     var name: String = ""
         set(value) {
             field = value
-            initials.setInfo(name, email)
+            initials.setInfo(name, email, backgroundColor)
         }
     var email: String = ""
         set(value) {
             field = value
-            initials.setInfo(name, email)
+            initials.setInfo(name, email, backgroundColor)
+        }
+    @ColorInt
+    var backgroundColor: Int? = null
```

Comment:
Let's name this `avatarBackgroundColor` since it ultimately gets propagated to `PersonaView` as well.

Reply:
And can you move it below the `avatarImageUri` property? Just want the image setting properties to have more priority. You have it in that location in other places, too.